### PR TITLE
[all] Adding platform support table for each individual plugin's README.md

### LINKED ISSUE #111: Add supported platforms section to README.md of the packages
## Use case
If you search on pub.dev and you look the `README.md` of the packages, it is never 100% clear, which platforms are supported. Only the top level `README.md` of the `plus_plugins` has this overview. But if you are on pub.dev you don't see the top level `README.md` of the `plus_plugions` git repo.

## Proposal
My proposal is to add the supported platforms tables from the top level `README.md` to the specifics `README.md` of the packages, like:

image

### PR DESCRIPTION
**Pull request by pr-Mais**
## Description

Adding platform support table for each individual plugin's README.md.

## Related Issues

Closes #111

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/battery_plus/README.md [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 A Flutter plugin to access various information about the battery of the device the app is running on.
 
+## Platform Support
+
+| Android | iOS | MacOS | Web | Linux | Window |
+|:-------:|:---:|:-----:|:---:|:-----:|:------:|
+|    ✔️    |  ✔️  |   ✔️   |  ✔️  |   ✔️   |    ✔️   |
```
Comment (miquelbeltran) [inline packages/battery_plus/README.md:13]: :mag:  Empty line missing

File: packages/network_info_plus/README.md [hunk 2]
```diff
@@ -7,6 +7,13 @@
 This plugin allows Flutter apps to discover network info and configure
 themselves accordingly.
 
+## Platform Support
+
+| Android | iOS | MacOS | Web | Linux | Window |
+|:-------:|:---:|:-----:|:---:|:-----:|:------:|
+|         |     |       |     |       |        |
```
Comment (miquelbeltran) [inline packages/network_info_plus/README.md:14]: :mag:  Extra line here

**Review by miquelbeltran — APPROVED** [review]
Thank you! I just pointed a couple of nitpicks which could be fixed before merge and that's it.

**Comment by pr-Mais** [comment]
Welcome! fixed them, the extra line was in the big README as well.

**Review by miquelbeltran — APPROVED** [review]

**Pull request merged**